infrastructure-playbooks/rolling_update.yml

```diff
@@ -1033,6 +1033,36 @@
         - import_role:
             name: ceph-dashboard
 
+- name: switch any existing crush buckets to straw2
+  hosts: "{{ mon_group_name | default('mons') }}[0]"
+  become: true
+  any_errors_fatal: true
+  gather_facts: false
+  tasks:
+    - name: set_fact ceph_cmd
+      set_fact:
+        ceph_cmd: "{{ hostvars[groups[mon_group_name][0]]['container_binary'] + ' run --rm --net=host -v /etc/ceph:/etc/ceph:z --entrypoint=ceph ' + ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment | bool else 'ceph' }}"
```

Comment:
No need to import_role ceph-default and ceph-facts (tasks_from container_binary.yml) here ?